### PR TITLE
1376 cannot load a spatial structures or a reactions bb

### LINKED ISSUE #1376: Cannot load a spatial structures BB
From the same project as in https://github.com/Open-Systems-Pharmacology/MoBi/issues/1354

1. Save the spatial structure as PKML
2. Create a new module and try to load it as PKML:

Application:
MoBi® 12.0.263

Unable to find a serializer for 'OSPSuite.Core.Domain.Builder.SpatialStructure'

Stack trace:
```
at OSPSuite.Serializer.Xml.XmlSerializerRepository`1.SerializerFor(Type typeOfObjectToSerialize)
   at MoBi.Core.Serialization.Xml.Services.XmlSerializationService.serializeFor(Type type)
   at MoBi.Core.Serialization.Xml.Services.XmlSerializationService.ElementNameFor(Type type)
   at MoBi.Presentation.Tasks.SerializationTask.retrieveElementsToDeserializeFromFile[T](XElement xelRoot, String fileName)
   at MoBi.Presentation.Tasks.SerializationTask.LoadMany[T](String fileName, Boolean resetIds)
   at MoBi.Presentation.Tasks.Interaction.InteractionTask.LoadItems[T](String filename)
   at MoBi.Presentation.Tasks.Interaction.InteractionTasksForModule.loadBuildingBlock(BuildingBlockType buildingBlockType, String filename)
   at MoBi.Presentation.Tasks.Interaction.InteractionTasksForModule.loadBuildingBlocksToModule(Module module, Func`1 getFilename)
   at MoBi.Presentation.Tasks.Interaction.InteractionTasksForModule.LoadBuildingBlocksToModule(Module module)
   at MoBi.Presentation.UICommand.LoadBuildingBlocksToModuleUICommand.PerformExecute()
   at OSPSuite.Presentation.UICommands.ObjectUICommand`1.Execute()
   at OSPSuite.Presentation.UICommands.UICommandExtensions.ExecuteWithinExceptionHandler(IUICommand uiCommand, IEventPublisher eventPublisher, IChangePropagator changePropagator)
```

### PR DESCRIPTION
**Pull request by rwmcintosh**
Fixes #1376

# Description

## Type of change

Please mark relevant options with an `x` in the brackets.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Integration tests
- [ ] Unit tests
- [x] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):

# Questions (if appropriate):

### CODE REVIEW

File: src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs
Comment (rwmcintosh) [inline src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs:1]: The real code changes are in this file

**Comment by rwmcintosh** [comment]
tiny code changes along with a core update

File: src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs [hunk 1]
```diff
@@ -153,13 +153,13 @@ private List<IBuildingBlock> loadBuildingBlock(BuildingBlockType buildingBlockTy
          switch (buildingBlockType)
          {
             case BuildingBlockType.Reaction:
-               items.AddRange(InteractionTask.LoadItems<ReactionBuildingBlock>(filename));
+               items.AddRange(InteractionTask.LoadItems<MoBiReactionBuildingBlock>(filename));
```
Comment (msevestre) [inline src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs:156]: if they are sent from PKSim, they are not from type MoBiReactionbUildingblock .dOes this still work?

Comment (rwmcintosh) [reply src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs:156]: Yes, I checked with a PK-Sim simulation.

Comment (rwmcintosh) [reply src/MoBi.Presentation/Tasks/Interaction/InteractionTasksForModule.cs:156]: I think this is why/how
```
public MoBiSpatialStructureXmlSerializer() : base(AppConstants.XmlNames.SpatialStructure)
```

**Pull request merged**